scene/2d/tile_map.compat.inc

```diff
@@ -34,8 +34,24 @@ Rect2i TileMap::_get_used_rect_bind_compat_78328() {
 	return get_used_rect();
 }
 
+void TileMap::_set_quadrant_size_compat_81070(int p_quadrant_size) {
+	set_rendering_quadrant_size(p_quadrant_size);
+}
+
+int TileMap::_get_quadrant_size_compat_81070() const {
+	return get_rendering_quadrant_size();
+}
+
+void TileMap::_force_update_compat_81070(int p_layer) {
+	notify_runtime_tile_data_update();
```

Comment:
`notify_runtime_tile_data_update()` takes a layer argument. Should `p_layer` be passed along here?

Reply:
Ah, my bad, that's an oversight.